bedrock-devnet/devnet/__init__.py

```diff
@@ -184,7 +184,8 @@ def devnet_l1_genesis(paths):
         log.info('Start to fork the public network. Wait to warm up the fork public network.')
         geth = subprocess.Popen([
             'anvil', '--fork-url', paths.l1_rpc_url, '--fork-block-number', str(paths.from_block_number),
-            '--chain-id', '1337'
+            '--chain-id', '1337', '--disable-block-gas-limit',
```

Comment:
Could you change the chainId to 900? When i tested the L1 contract upgrade on Devnet, I had to manually change the deployment/devnetL1/.chainId to 900 because the chain ID was 1337. 
It's not a big deal, but for reduce confusion in the future, i think it is good change it